crates/apub/src/objects/note_wrapper.rs

```diff
@@ -32,12 +32,12 @@ impl Object for ApubNote {
     _object_id: Url,
     _context: &Data<Self::DataType>,
   ) -> LemmyResult<Option<Self>> {
-    unimplemented!()
+    Err(LemmyErrorType::Unknown("not implemented".to_string()).into())
```

Comment:
I looked at some of the others, and these all have specific `FederationErrors` for actions that can't be done. But its up to you there.

Also worth looking into why the unimplemented and todo passed the CI.